quasar-core/src/main/java/co/paralleluniverse/concurrent/util/ThreadAccess.java

```diff
@@ -21,6 +24,8 @@
 import java.security.*;
 import java.util.*;
 
+import static java.security.AccessController.doPrivileged;
+
 /**
  *
  * @author pron
```

Comment:
Shouldn't you also update the `@author` tags? This is not a trivial change...

Reply:
To quote Admiral Akbar: "It's a trap!"